src/frontend/screens/GpsModal.tsx

```diff
@@ -0,0 +1,183 @@
+    id: 'screens.GpsModal.locationDD',
+    defaultMessage: 'Coordinates Decimal Degrees',
+    description: 'Section title for DD coordinates',
+  },
+  details: {
+    id: 'screens.GpsModal.details',
+    defaultMessage: 'Details',
+    description: 'Section title for details about current position',
+  },
+  yes: {
+    id: 'screens.GpsModal.yes',
+    defaultMessage: 'Yes',
+    description: 'if a location sensor is active yes/no',
+  },
+  no: {
+    id: 'screens.GpsModal.no',
+    defaultMessage: 'No',
+    description: 'if a location sensor is active yes/no',
+  },
+  locationSensors: {
+    id: 'screens.GpsModal.locationSensors',
+    defaultMessage: 'Sensor Status',
+    description: 'Heading for section about location sensor status',
+  },
+});
+
+const GpsModalRow = ({label, value}: {label: string; value: string}) => (
+  <View style={styles.row}>
+    <Text style={styles.rowLabel}>{label}</Text>
+    <Text style={styles.rowValue}>{value}</Text>
+  </View>
+);
+
+export const GpsModal = () => {
+  const {location} = useLocation({maxDistanceInterval: 0});
+  const lastKnownLocationQuery = useLastKnownLocation();
+  const provider = useLocationProviderStatus();
+  const {coordinateFormat} = usePersistedSettings();
+  const {formatMessage: t} = useIntl();
+
+  const locationTimestamp =
+    location?.timestamp || lastKnownLocationQuery.data?.timestamp;
+
+  return (
+    <ScrollView style={styles.container} testID="gpsScreenScrollView">
+      <View style={styles.infoArea}>
+        <Text style={styles.sectionTitle}>
+          <FormattedMessage {...m.lastUpdate} />
+        </Text>
+        <DateDistance
+          style={styles.rowValue}
+          date={locationTimestamp ? new Date(locationTimestamp) : new Date()}
+        />
+        {location && (
+          <>
+            <Text style={styles.sectionTitle}>
+              <FormattedMessage {...m[coordinateFormat]} />
+            </Text>
+            <Text style={styles.rowValue}>
+              <FormattedCoords
+                lon={location.coords.longitude}
+                lat={location.coords.latitude}
+                format={coordinateFormat}
+              />
+            </Text>
+            <Text style={styles.sectionTitle}>
+              <FormattedMessage {...m.details} />
+            </Text>
+            {Object.entries(location.coords).map(([key, value]) => (
+              <GpsModalRow
+                key={key}
+                label={key}
+                value={typeof value === 'number' ? value.toFixed(5) : ''}
+              />
+            ))}
+          </>
+        )}
+        {provider && (
+          <>
+            <Text style={styles.sectionTitle}>
+              <FormattedMessage {...m.locationSensors} />
+            </Text>
+            {Object.entries(provider).map(([key, value]) => (
+              <GpsModalRow
+                key={key}
+                label={key}
+                value={value != null ? t(m.yes) : t(m.no)}
+              />
```

Comment:
Some of these values are showing up as "Yes" when they shouldn't be.

I have my Location Services Turned off turned off in this picture:

<img width="289" alt="image" src="https://github.com/digidem/comapeo-mobile/assets/67773827/deee3453-ce24-4a3c-a8db-aa0bed71c4b4">

Reply:
i remember running into this - forgot to document as a known issue. will note that i get this same behavior on Mapeo Mobile, so my guess is that there's something about the location hooks that providing stale data?

Reply:
> forgot to document as a known issue.

ah technically the last screenshot in my description shows this but i never highlighted it as an issue 😄 but yeah, given that the behavior is consistent with Mapeo Mobile (despite being wrong), I'd be okay with filing an issue and fixing as a follow-up (unless the fix is super easy)

Reply:
yeah! lets merge this then. Can you create an issue for this?

Reply:
yeah best guess is that there's no API from expo-location that will react to location services being turned off (usually done via settings outside the app)

in [`useLocationProviderStatus()`], the provider status is only refetched if the permissions change, which isn't the case in this scenario

Reply:
ah wait we have polling set up though, so it should be refetching anyways 🤔 going to dive deeper right now then...

Reply:
ah turns out this is just a bug from the ported code 😅

```suggestion
            {Object.entries(provider).map(([key, value]) => (
              <GpsModalRow
                key={key}
                label={key}
                value={value ? t(m.yes) : t(m.no)}
              />
```

Reply:
fixed via https://github.com/digidem/comapeo-mobile/pull/212/commits/5ab28c984d005b10091b6c69227d6f52ac0dc832